examples/ethernet/full/client.toit

```diff
@@ -8,9 +8,13 @@ A simple client for an Ethernet network.
 Requires the Ethernet provider to be installed.
 */
 
+import net
 import net.ethernet
 
+use network/net.Client:
```

Comment:
I would probably put this after main everywhere.

Reply:
done.